crates/trie/trie/src/prefix_set.rs

```diff
@@ -143,15 +155,20 @@ impl PrefixSetMut {
 /// See also [`PrefixSetMut::freeze`].
 #[derive(Debug, Default, Clone)]
 pub struct PrefixSet {
-    keys: Arc<Vec<Nibbles>>,
     index: usize,
+    all: bool,
```

Comment:
can we rename this to make it obvious what all means.

like all_keys_changed or similar

Reply:
gotta a doc comment here, but gonna argue the name change. prefix set is the set of changed keys. so, it is implied and unsure how `PrefixSet::all` could be misunderstood. feel free to persuade otherwise

Reply:
can you also note that when this is set, index should be zero and `keys` should be empty?